plugins/base/src/main/kotlin/resolvers/external/javadoc/JavadocExternalLocationProvider.kt

```diff
@@ -30,9 +30,13 @@ open class JavadocExternalLocationProvider(
         if (classNames == null) {
             return "$docWithModule$packageLink/package-summary$extension".htmlEscape()
         }
+        val classSplit = classNames?.split(".").orEmpty()
+        val enumEntityAnchor = classSplit.takeIf { classSplit.size > 1 }?.get(1)
```

Comment:
Not only enums can have dot in class names but also inner classes:
https://docs.oracle.com/javase/8/docs/api/java/rmi/activation/ActivationGroupDesc.CommandEnvironment.html

Reply:
Then I suggest adding a some `extra` in DRI for enum entry, e.g. "enum entry" (DRI extra is a text, it's weird).

Reply:
From where would you like to get this information?

Reply:
in `DRIFactory.kt`

Reply:
i wasn't talking about a place in code where to do it, i was more concerned about the information that we have available for example in `@see` tag: `@see java.nio.file.StandardCopyOption.REPLACE_EXISTING`

Reply:
If I got it correctly, a link to entities can be in `@see`, `@throws`, `@exception` and MD links. But the information we get from "kotlin analysis" descriptors.

Reply:
So i've checked out this branch:
- it is quite bad that we have different DRI-s for the same link depending on the language that we run over. For example in java the link syntax is: `@see java.nio.file.StandardCopyOption#REPLACE_EXISTING` whereas in kotlin `java.nio.file.StandardCopyOption.REPLACE_EXISTING`. This results in different dri-s being created (kotlin as a dot in classnames but java has REPLACE_EXISTING as callable). 

- I think we are forced to include something in the extras of DRI as i realy dont want to change it's structure for compatibility reasons. Imho kotlin the way it is represented in kotlin is ok. I've debugged there and we have a type of the entity in analysis so adding this extra should be easy. 

Format of this extra is up to you